app/models/page.rb

```diff
@@ -11,4 +11,14 @@ class Page < ApplicationRecord
     indexes :base_path, type: "text", analyzer: "english" do
     end
   end
+
+  def self.unique_visitors_for_phrase(phrase, start_date, end_date, sort_key: "unique_visitors", sort_dir: "desc")
+    date_range = start_date..end_date
+
+    Page.joins(page_visits: [{ visit: [{ survey_visit: [{ survey: [{ survey_answers: [{ mentions: :phrase }] }] }] }] }])
```

Comment:
WOW! I have a feeling this could cause performance problems down the line but it's ok for the moment

Reply:
We may have to possibly rethink our database schema, if we do find performance problems further down the road.

Reply:
yup, as we're still learning it's fine but a good thing to have in the back of our minds 💯

Reply:
Absolutely.